packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -147,6 +147,12 @@ class BottomNavigationBar extends StatefulWidget {
   /// The [iconSize], [selectedFontSize], [unselectedFontSize], and [elevation]
   /// arguments must be non-null and non-negative.
   ///
+  /// The [selectedLabelStyle.color] and [unselectedLabelStyle.color] values
+  /// will be ignored in favor of [selectedItemColor] and [unselectedItemColor].
+  ///
+  /// If both [selectedLabelStyle.fontSize] and [selectedFontSize] are set,
+  /// [selectedLabelStyle.fontSize] will be used.
+  ///
```

Comment:
Any way we could be consistente between these two policies and always let the specifc one win (selectedFontSize)?

Reply:
Yes Updated so that `selectedTextStyle.color` is taken over `selectedItemColor`.